src/components/Reviews.jsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react"
+function Reviews () {
+    const [reviews, setReviews] = useState([])
```

Comment:
We also want to be having a loading state, so if we are still awaiting the data, we will show a loading message. Take a look at the notes to see how to do this.